modules/support-must-gather-targeted-collection-gathering-data.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-must-gather-targeted-collection-gathering-data_{context}"]
+= Gathering data for specific resources
+
+Instead of collecting data for the entire cluster, you can direct `must-gather` to inspect a specific resource. This is highly effective for isolating issues within a single project, operator, or application.
```

Comment:
```suggestion
Instead of collecting data for the entire cluster, you can direct the `must-gather` tool to inspect a specific resource. This method is highly effective for isolating issues within a single project, Operator, or application.
```

The word Operator must be uppercase. See [Operator capitalization](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization).